### PR TITLE
Add mqtt events

### PR DESCRIPTION
**Pull request by ikstream**
Inform user via mqtt about powquty events. 
The structure is 
DEV UUID, Event, Timestamp
BERTUB001,Voltage dip < 10%,1499556605,1748

If the appearance of events is to close to each other mqtt can not notify the user
at the moment.

### CODE REVIEW

File: luci-app-powquty/luasrc/model/cbi/powquty/powquty.lua [hunk 1]
```diff
@@ -36,6 +36,18 @@ mqtt_topic = s:taboption("mqtt", Value, "mqtt_topic", "mqtt_topic", "The topic u
...
 
+mqtt_event_host = s:taboption("mqtt", Value, "mqtt_event_host", "mqtt_event_host", "IP-address or URL to the MQTT broker who receives the publish message of powquty events")
+mqtt_event_host.datatype = "string"
+mqtt_event_host.default = "localhost"
+
+mqtt_event_topic = s:taboption("mqtt", Value, "mqtt_event_topic", "mqtt_event_topic", "The topic under which powquty will publish EN50160 events")
+mqtt_event_topic.datatype = "string"
+mqtt_event_topic.default = "device/en50160-event"
+
+mqtt_event_flag = s:taboption("mqtt", Flag, "mqtt_event_flag", "mqtt_event_flag", "If activated powquty will send notifications via mqtt on EN50160 events")
```
Comment (thuehn) [inline luci-app-powquty/luasrc/model/cbi/powquty/powquty.lua]: better: about EN50160 events

Comment (ikstream) [inline luci-app-powquty/luasrc/model/cbi/powquty/powquty.lua]: reworded both occasions

**Comment by thuehn** [comment]
thx

**Comment by thuehn** [comment]
canyou solve this merge conf ?

**Comment by thuehn** [comment]
thx

**Pull request merged**